RUNNING_DOCKER.RST

```diff
@@ -0,0 +1,369 @@
+
+If you have not set up a connected app please visit the following to achieve this.
+https://cumulusci.readthedocs.io/en/latest/tutorial.html#creating-a-connected-app
+Once done, confirm with the aformentioned command to ensure you are properly configured 
+with regards to the connected_app variables.
+
+
+Production Variables
+--------------------
+
+GITHUB_WEBHOOK_SECRET
+    definition here
+DJANGO_AWS_ACCESS_KEY_ID
+    definition here
+DJANGO_AWS_SECRET_ACCESS_KEY
+    definition here
+DJANGO_AWS_STORAGE_BUCKET_NAME
+    definition here
+DJANGO_SERVER_EMAIL
+    definition here
+DJANGO_SENTRY_DSN
+    definition here
+
+
+Other Variables (some preset for user)
+--------------------------------------
+
+BUILD_ENV: 
+    Environment variable used to determine what dependencies and scripts to run when 
+    installing dependencies and populating databases, currently set in docker-compose.yml
+    web service environment variable
+
+NODE_VERSION: 
+    Environment variable used to set node version for download, this variable is set in the Dockerfile
+
+YARN_VERSION: 
+    Environment variable used to set yarn version for download, this variable is set in the Dockerfile
+
+PYTHONUNBUFFERED: 
+    Environment variable set in Dockerfile used to not write .pyc files to Docker container
+       
+DATABASE_URL:
+    Environment variable set in Dockerfile. Represents the full path of database url.
+
+REDIS_URL: 
+    This represents the url to the location where the redis server, configured for Meta CI. Set in Dockerfile.
+
+DJANGO_HASHID_SALT: 
+    This represents the hashid salt for the django application, currently set to 
+    arbritary string due to non production defaults, can be overridden 
+    in docker-compose.yml. Currently set in Dockerfile.
+
+DJANGO_SECRET_KEY: 
+    This represents the key for the django application, currently set to arbritary
+    string due to non production defaults, can be overridden in docker-compose.yml.
+    Currently set in Dockerfile.
+
+Building Your Docker Containers:
+-------------------------------
+
+This next section assumes you have downloaded ``docker`` and ``docker-compose``.
+Additionally it assumes you have a ``.env`` file in the root directory of this 
+project, a template of variables needed can be found under ``env.example``.
+
+To configure and run your environment you must run 2 commands in the root directory of MetaCI
+Note that docker-compose build will take some significant time to build the first time but will
+be much faster for subsequent builds. It is also important to note that once you bring 
+up the web application it will take roughly 60 seconds to fully compile. 
+
+::
+    
+    docker-compose build
+
+Running Your Docker Containers:
+-------------------------------
+If you would like to populate your instance of MetaCI with test data simply 
+set DJANGO_SETTINGS_MODULE to config.settings.local in the ``.env`` docker-compose.yml file. 
```

Comment:
This shouldn't be necessary; `config.settings.local` is already set as the default in the Dockerfile

Reply:
Do you like my explanation now? It also is making me consider revising the logic for utility/start_server.sh

Reply:
@jofsky no it still doesn't make sense to me. Setting DJANGO_SETTINGS_MODULE to config.settings.local in `.env` won't change anything because it's already set to that value in the Dockerfile.

Reply:
its set in docker-compose?

Reply:
@jofsky it's set in Dockerfile: https://github.com/SFDO-Tooling/MetaCI/blob/feature/docker/Dockerfile#L32